test/el-test.js

```diff
@@ -126,3 +127,129 @@ describe('yo polymer:el --path test', function () {
     );
   });
 });
+
+describe('yo polymer:el TDD test', function () {
+
+  before(function (done) {
+    helpers.run(path.join(__dirname, '../el'))
+      .inDir(path.join(__dirname, './tmp'))
+      .on('ready', function (generator) {
+        var js = "WCT.loadSuites(['my-greeting-basic.html', 'my-list-basic.html']);";
+        fs.outputFileSync(path.join(__dirname, './tmp/app/test/index.html'), js);
```

Comment:
I'm wondering if it would be better to run the `yo polymer:app` generator first with `--skip-install` flags so you're testing against the actual index.html file. Here's an example of running both (https://github.com/yeoman/generator-polymer/blob/master/test/el-test.js#L9-L27). The reason I'm suggesting this is because I feel like if anything is going to break, it's probably going to be from regexing around inside that file, so testing with something as close to the real thing may be valuable.